core/src/main/scala/kafka/utils/ZkUtils.scala

```diff
@@ -151,10 +151,6 @@ object ZkUtils {
     Json.legacyEncodeAsString(Map("version" -> 1, "brokerid" -> brokerId, "timestamp" -> timestamp.toString))
   }
 
-  def preferredReplicaLeaderElectionZkData(partitions: scala.collection.Set[TopicAndPartition]): String = {
```

Comment:
`ZkUtils` was used by a few components back in the day, but it is currently deprecated. I think I'd probably leave this API around even if it is not used. All of `ZkUtils` will go away before too long anyway.

Reply:
Okay. Good to know. I missed the `deprecated` notice.